### PR TITLE
fix: individually tally votes

### PR DESCRIPTION
**Pull request by axejintao**
# Summary 

Individually counts submitted vote weights

### CODE REVIEW

File: src/citadel/citadel.utils.ts [hunk 1]
```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
   const client = new GraphQLClient(CITADEL_KNIGHTS_ROUND_SUBGRAPH_URL);
   const graphSdk = setKnightsRoundStatsSdk(client);
 
-  const knightsRoundStatResp = await graphSdk.KnightsRounds();
+  let totalVotes: VoteFragment[] = [];
+
+  let voteId;
+  while (true) {
+    const voteData = await graphSdk.Votes({
+      first: 2,
```
Comment (axejintao) [inline src/citadel/citadel.utils.ts:243]: will remove this subsequntly just want to verify iteration in staging w/ low vote counts

File: src/citadel/citadel.utils.ts [hunk 2]
```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
...
+      orderBy: Vote_OrderBy.Id,
+      orderDirection: OrderDirection.Asc,
+      where: {
+        id_gt: voteId,
+      },
+    });
+    const { votes } = voteData;
+    if (votes.length === 0) {
+      break;
+    }
```
Comment (NamelsKing) [inline src/citadel/citadel.utils.ts:253]: i think here should be 
```js
voteId = votes[votes.length - 1].id;
```

File: src/citadel/citadel.utils.ts [hunk 3]
```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
...
-    votes: knight.voteCount,
-    voteWeight: (Math.pow(knight.voteAmount, 1 / 1.3) / totalWeight) * 100,
-    votersCount: knight.voters.length,
-    funding: formatBalance(knight.voteAmount, ctdlToken.decimals) * CITADEL_START_PRICE_USD,
-  }));
+  return Object.entries(knightVoteWeights).map((e) => {
+    const [knight, { voteCount, voteWeight }] = e;
+    return {
+      knight: getKnightEnumByIx(Number(knight)),
+      votes: voteCount,
```
Comment (NamelsKing) [inline src/citadel/citadel.utils.ts:291]: i believe here u have `voteAmount` in CTDL, not count of `votes`

Comment (axejintao) [reply src/citadel/citadel.utils.ts:291]: @mrbasado please confirm here, discussed offline but I think we do want the 'votes' to be the fully tallied amounts of CTDL purchased while the "vote Count" is the number of users that contributed those votes

Comment (mrbasado) [reply src/citadel/citadel.utils.ts:291]: yeah, votes = total xctld, votercount = number of people, vote weight is thier share of supply (for the freefor all part)

File: src/citadel/citadel.utils.ts [hunk 4]
```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
...
```
Comment (mrbasado) [inline src/citadel/citadel.utils.ts:291]: Lets rename this field to voteAmount, or totalVoteAmount, etc to be more clear

**Review by mrbasado — DISMISSED** [review]

File: src/config/constants.ts [hunk 5]
```diff
@@ -49,7 +49,7 @@ export const BLOCKNATIVE_API_KEY = getEnvVar('BLOCKNATIVE_API_KEY');
 export const DEFAULT_PAGE_SIZE = 20;
 
 export const swaggerConfig: SwaggerSettings = {
-  path: '/docs',
+  path: 'docs',
```
Comment (NamelsKing) [inline src/config/constants.ts:52]: not sure, but it can break `swagger` >_>

**Review by NamelsKing — DISMISSED** [review]

**Pull request closed without merging**